android/app-example/src/main/java/com/badoo/ribs/example/rib/hello_world/builder/HelloWorldComponent.kt

```diff
@@ -17,7 +18,8 @@ internal interface HelloWorldComponent {
     interface Factory {
         fun create(
             dependency: HelloWorld.Dependency,
-            @BindsInstance customisation: HelloWorld.Customisation
+            @BindsInstance customisation: HelloWorld.Customisation,
+            @BindsInstance savedInstanceState: Bundle?
```

Comment:
Do we have any problems with ribs tree unwrapping it?
Or do the nodes refer to the bundle just by id?

Reply:
Not sure what you mean. Node, Interactor, Router stores their own bundles nested inside this top level one. Router has ConfigurationFeature, which stores its state (and thus the bundles of all children) inside Router's bundle. So it's all nested.